Lps 89242 Remove unnecessary mocks in portal-workflow-kaleo-runtime-integration-impl

lanceji92674957 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

lanceji92674957 commented:
ci:test:relevant

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 21 out of 24 jobs passed in 1 hour 23 minutes 42 seconds 949 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/96dcfdb6a9b4f0b79cbfbf5d7e26ff7bd498ce25">96dcfdb6a9b4f0b79cbfbf5d7e26ff7bd498ce25</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/43bcfb019eb0f150dc6bea243624c16c03a011aa">43bcfb019eb0f150dc6bea243624c16c03a011aa</a></p>21 out of 24 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest(master)/3680/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275820/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</strong></strike></a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/138481/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li></ul></div><details><summary><strong>21 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/187107/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/138480/">test-portal-acceptance-pullrequest-batch(master)/functional-smoke-tomcat90-mysql57-open11/0</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/187108/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/138482/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/285927/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275821/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272515/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/138483/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/187109/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275822/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/285928/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272646/">test-portal-acceptance-pullrequest-batch(master)/lpkg-base-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265381/">test-portal-acceptance-pullrequest-batch(master)/lpkg-controller-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/265382/">test-portal-acceptance-pullrequest-batch(master)/modules-compile-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/264037/">test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272647/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/284167/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/187110/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-project-templates-jdk8</a></li><li><a href="https://test-1-20.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/285929/">test-portal-acceptance-pullrequest-batch(master)/pmd-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272516/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/272648/">test-portal-acceptance-pullrequest-batch(master)/service-builder-jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-16.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3680/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/275820/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></h5><div><h6>Job Results:</h6><p>6 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/275820//consoleText">AXIS_VARIABLE=0,label_exp=!master #275820</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/275820//testReport/com.liferay.portal.modules/ModulesStructureTest/testScanBuildScripts">ModulesStructureTest.testScanBuildScripts</a><pre><code>junit.framework.AssertionFailedError: Missing modules/apps/document-library/document-library-rest-impl/build.gradle
	at com.liferay.portal.modules.ModulesStructureTest$1.preVisitDirectory(ModulesStructureTest.java:232)
	at com.liferay.portal.modules.ModulesStructureTest$1.preVisitDirectory(ModulesStructureTest.java:123)
	at java.nio.file.Files.walkFileTree(Files.java:2677)
	at java.nio.file.Files.walkFileTree(Files.java:2742)
	at com.liferay.portal.modules.ModulesStructureTest.testScanBuildScripts(ModulesStructureTest.java:121)
</code></pre></div></li></ol></div></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-1.liferay.com/job/test-portal-acceptance-upstream(master)">acceptance upstream results</a> at a7820f8b29302ad2a117efbd7eec562c0a62044c:</strong></summary><ol><li><div><h5><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/138481/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>4 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/138481//consoleText">AXIS_VARIABLE=0,label_exp=!master #138481</a><ol><li><div><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/138481//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowadminCPWebcontent_ApproveInWCSubfolderWorkflow_">LocalFile.WorkflowadminCPWebcontent#ApproveInWCSubfolderWorkflow</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1548459140474/test-portal-acceptance-pullrequest(master)/3680/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WorkflowadminCPWebcontent_ApproveInWCSubfolderWorkflow/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1548459140474/test-portal-acceptance-pullrequest(master)/3680/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.WorkflowadminCPWebcontent_ApproveInWCSubfolderWorkflow/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-16/1548459140474/test-portal-acceptance-pullrequest(master)/3680/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol></details></details></html>

mtambara committed:
LPS-90208 Move spi loading from arquillian-container-test-impl-base

mtambara committed:
LPS-90208 Remove unused

mtambara committed:
LPS-90208 Use ours

mtambara committed:
LPS-90208 Sort

mtambara committed:
LPS-90208 Inline ContainerTestExtension

mtambara committed:
LPS-90208 Copy DeploymentGenerator

mtambara committed:
LPS-90208 Grab container directly

mtambara committed:
LPS-90208 Only one deployment

mtambara committed:
LPS-90208 Don't need check

mtambara committed:
LPS-90208 Don't have any AuxiliaryArchiveProcessors

mtambara committed:
LPS-90208 We know it's a ClassContainer

mtambara committed:
LPS-90208 ProtocolDefinition is available

mtambara committed:
LPS-90208 Only protocol is JMXOSGiProtocol

mtambara committed:
LPS-90208 BndDeploymentScenarioGenerator only generates one DeploymentDescription

mtambara committed:
LPS-90208 Inline

mtambara committed:
LPS-90208 Only have one deployment

mtambara committed:
LPS-90208 No ApplicationArchiveProcessors

mtambara committed:
LPS-90208 Rename

mtambara committed:
LPS-90208 Don't need to validate

mtambara committed:
LPS-90208 Grab container directly

mtambara committed:
LPS-90208 Inline

mtambara committed:
LPS-90208 Copy ContainerEventController

mtambara committed:
LPS-90208 Use BndDeploymentScenarioGenerator directly

mtambara committed:
LPS-90208 Just listen for TestEvent

mtambara committed:
LPS-90208 Copy RemoteTestExecuter

mtambara committed:
LPS-90208 Directly use JMXOSGiProtocol

mtambara committed:
LPS-90208 Inline ContainerMethodExecutor

mtambara committed:
LPS-90208 Inline

mtambara committed:
LPS-90208 Listen for all test event, they're all executed remotely

mtambara committed:
LPS-90208 Remove JMXOSGiProtocol

mtambara committed:
LPS-90208 Squash DeploymentGenerator into ContainerEventController

mtambara committed:
LPS-90208 Don't need to create anymore

mtambara committed:
LPS-90208 Rename

mtambara committed:
LPS-90208 Remove unused events

mtambara committed:
LPS-90208 Squash ContainerLifecycleController into ContainerEventController

mtambara committed:
LPS-90208 Direct calls

mtambara committed:
LPS-90208 Inline

mtambara committed:
LPS-90208 Don't need to fire events

mtambara committed:
LPS-90208 Invert if

mtambara committed:
LPS-90208 Squash ContainerDeployController into ContainerEventController

mtambara committed:
LPS-90208 These don't need to be produced

mtambara committed:
LPS-90208 Don't need DeploymentScenario

mtambara committed:
LPS-90208 Already gets added in BndDeploymentScenarioGenerator

mtambara committed:
LPS-90208 Remove arquillian-container-test-impl-base

mtambara committed:
LPS-90208 Not thrown

mtambara committed:
LPS-90208 Don't need to make anymore

mtambara committed:
LPS-90208 Don't need HTTPContext

mtambara committed:
LPS-90384 Don't need this ContainerDef

mtambara committed:
LPS-90384 Don't need to negate

mtambara committed:
LPS-90384 Remove arquillian-config-impl-base

shuyangzhou committed:
LPS-90208 This is never used.

shuyangzhou committed:
LPS-90208 Squash Container creation into ContainerEventController

shuyangzhou committed:
LPS-90208 Exchange MBeanServerConnection directly.

lanceji92674957 committed:
LPS-89242 No need to mock

javiergamarra committed:
LPS-89787 extract User transformer to Util class

javiergamarra committed:
LPS-89787 migrate categories API

javiergamarra committed:
LPS-89787 add dependendy

javiergamarra committed:
LPS-89787 sf

alejandrohdezma committed:
LPS-89795 Implement Keyword#get

alejandrohdezma committed:
LPS-89795 manually add delete method (should be generated)

alejandrohdezma committed:
LPS-89795 Implement Keyword#delete

alejandrohdezma committed:
LPS-89795 Implement Keyword#post

alejandrohdezma committed:
LPS-89795 Implement Keyword#put

alejandrohdezma committed:
LPS-89795 Control AssetTag exceptions

This kind of exceptions should not be controlled in a generic way in an ExceptionMapper, since that may provoke loosign the error semantics

shuyangzhou committed:
LPS-90208 Inline

shuyangzhou committed:
LPS-90208 No reason to assign to local var

shuyangzhou committed:
LPS-90208 Container is always created by BeforeSuite event

shuyangzhou committed:
LPS-90208 Stop using  DeploymentScenarioGenerator

shuyangzhou committed:
LPS-90208 Unnecessary state verification, we do the state transition, we can count on ourselves.

shuyangzhou committed:
LPS-90208 AfterSuite is only ever fired out on the server side. ContainerEventController only exists on client side, it is not possible for it to receive a AfterSuite event. Essentially JMX container does not need any kind of stop handling.

shuyangzhou committed:
LPS-90208 Let BndDeploymentDescriptionUtil fully preparing the DeploymentDescription

shuyangzhou committed:
LPS-90208 Stop using ContainerMethodExecutor

shuyangzhou committed:
LPS-90208 Squash RemoteTestExecuter into JMXMethodExecutor

shuyangzhou committed:
LPS-90208 We are programmingly creating LiferayRemoteDeployableContainer now.

javiergamarra committed:
LPS-90435 support for tags

brianchandotcom committed:
LPS-89787 Move it to a v1_0 dir

javiergamarra committed:
LPS-90435 add tags following standard

brianchandotcom committed:
LPS-89787 I don't understand why this is plural

brianchandotcom committed:
LPS-89787 I don't understand why this is plural

brianchandotcom committed:
LPS-89787 Rename

brianchandotcom committed:
LPS-89787 Simplify

brianchandotcom committed:
LPS-89787 Ensure our stuff is actually used

brianchandotcom committed:
LPS-89787 Simplify

joshchong committed:
LRQA-46741 Temporarily ignore

tinatian committed:
LPS-89456 Fix UpgradeContentTargeting, use right column name

jpince committed:
LPS-89658 Semver

brianchandotcom committed:
LPS-89787 auto SF

brianchandotcom committed:
LPS-89787 simplify

javiergamarra committed:
LPS-90435 regenerate missing APIs that didn't had a direct correlation with the schema

brianchandotcom committed:
LPS-89787 preallocate size

brianchandotcom committed:
LPS-89795 Simplify

javiergamarra committed:
LPS-90435 use plural for paths and singular for params

javiergamarra committed:
LPS-90435 don't ignore plural in methods, use path

javiergamarra committed:
LPS-90435 regenerate

javiergamarra committed:
LPS-90435 adapt to generated (generator is creating Page<Long> that does not compile)

brianchandotcom committed:
LPS-89795 Plural?

brianchandotcom committed:
LPS-89795 Sort

brianchandotcom committed:
LPS-90435 regen, at least get it to compile

patriciajeanperez committed:
LRQA-46736 Fix locator for bold and italic content

patriciajeanperez committed:
LRQA-46743 Fix locator for report button

patriciajeanperez committed:
LRQA-46738 Fix locator for styling dropdown

javiergamarra committed:
LPS-90435 modify generator to compile when creating Page<Long>

javiergamarra committed:
LPS-90435 generated

lanceji92674957 committed:
LPS-89242 Replace mock with wrapper

patriciajeanperez committed:
LRQA-44632 Fix locator for image thumbnail

lanceji92674957 committed:
LPS-89242 Extends KaleoTaskAssignmentInstanceLocalServiceWrapper to replace Mockito.verify for KaleoTaskAssignmentInstanceLocalService

lanceji92674957 committed:
LPS-89242 Inline

lanceji92674957 committed:
LPS-89242 Extends KaleoTaskInstanceTokenWrapper to replace Mockito.verify for KaleoTaskInstanceToken

lanceji92674957 committed:
LPS-89242 No need to have local variables
